apps/web/src/app/[locale]/challenge/_components/comments/comment-skeleton.tsx

```diff
@@ -1,15 +1,22 @@
+import { Skeleton } from '@repo/ui/components/skeleton';
+
 export function CommentSkeleton() {
   return (
-    <div className="flex flex-col space-y-10 px-4 py-3">
-      <div className="prose-invert prose-h3:text-xl flex flex-col gap-6 space-y-2">
-        <div className="h-6 rounded-lg bg-zinc-300 dark:bg-zinc-700" />
-        <div className="h-6 rounded-lg bg-zinc-300 dark:bg-zinc-700" />
-        <div className="h-6 rounded-lg bg-zinc-300 dark:bg-zinc-700" />
-        <div className="h-6 rounded-lg bg-zinc-300 dark:bg-zinc-700" />
-        <div className="h-6 rounded-lg bg-zinc-300 dark:bg-zinc-700" />
-        <div className="h-6 rounded-lg bg-zinc-300 dark:bg-zinc-700" />
-        <div className="h-6 rounded-lg bg-zinc-300 dark:bg-zinc-700" />
-      </div>
-    </div>
+    <>
+      {Array.from({ length: 3 }, (_, i) => i + 1).map((id) => (
```

Comment:
We can move this array creation outside of the JSX to prevent re-creating it on rerenders:

```tsx
const SKELETONS = Array.from({ length: 3 }, (_, i) => i + 1)

export function CommentSkeleton() {
  return (
    <>
      {SKELETONS.map((id) => (
```